Workbooks/WorkbooksMetadata.json

```diff
@@ -739,5 +739,44 @@
     "templateRelativePath": "SysmonThreatHunting.json",
     "subtitle": "",
     "provider": "Edoardo Gerosa"
+  },
+  {
+    "workbookKey": "VMwareCarbonBlackWorkbook",
+    "logoFileName": "vmwarecarbonblack_logo.svg",
+    "description": "Gain extensive insight into VMware Carbon Black Cloud - Endpoint Standard by analyzing, collecting and correlating Event logs.\nThis workbook provides visibility into Carbon Black managed endpoints and identified threat event",
+    "dataTypesDependencies": [ "CarbonBlackEvents_CL" ],
```

Comment:
You can add commas to separate multiple data types - I see Carbon Black has multiple tables, please include all in the dependencies. Same for Proofpoint and others as well - extend this feedback to all.

Reply:
corrections have been made